content/en/blog/_posts/2022-08-26-Announcing-Auto-Refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: blog 
+title: "Announcing the Auto-refreshing Official Kubernetes CVE
+Feed"
+date: 2022-08-26 
```

Comment:
Please edit the path of this file and also change this field to match the assigned publication date: 2022-09-12

Reply:
done